be/src/exec/olap_scan_node.cpp

```diff
@@ -1652,6 +1654,7 @@ void OlapScanNode::scanner_thread(OlapScanner* scanner) {
     // and transfer thread
     _scan_batch_added_cv.notify_one();
     _scan_thread_exit_cv.notify_one();
+    current_thread.update_mem_tracker(nullptr);
```

Comment:
The thread may be returned early.

Reply:
OK, Is combing where the threads start and end.
There is currently a problem that query memtracker cannot be deleted in time.